README.md

```diff
@@ -262,7 +262,7 @@ constrained support. The package was created with [Cookiecutter] and the
 ## Cite us
 Are you using PySwarms in your project or research? Please cite us!
 
-* Miranda L.J., (2018). PySwarms: a research toolkit for Particle Swarm Optimization in Python. *Journal of Open Source Software*, 3(21), 433, https://doi.org/joss.00433
+* Miranda L.J., (2018). PySwarms: a research toolkit for Particle Swarm Optimization in Python. *Journal of Open Source Software*, 3(21), 433, [https://doi.org/joss.00433](https://doi.org/10.21105/joss.00433)
```

Comment:
You can also put 10.21105 inside the square brackets 👍

Reply:
I need to make new PR? 

*I'm not sure how to collaborate. I'll make one. Sorry, noob here.